src/calva_power_tools/tool/clay.cljs

```diff
@@ -1,22 +1,22 @@
 (ns calva-power-tools.tool.clay)
 
 (def make-current-form-snippet
-  '(do (require '[scicloj.clay.v2.snippets])
+  '(do (clojure.core/require '[scicloj.clay.v2.snippets])
```

Comment:
I think that when `require` is not available, there are bigger problems. But let's try it this way and see what gives.

Reply:
Clay just needs to be called, so an empty namespace (which Calva provides) is enough 😄